pallets/schemas/src/types.rs

```diff
@@ -18,4 +17,6 @@ where
 	pub model: BoundedVec<u8, MaxModelSize>,
 	/// The payload location
 	pub payload_location: PayloadLocation,
+	/// grants for the schema
+	pub grants: Grants,
```

Comment:
I think we will need to create a new Version of Schema to be able to migrate between old version to new.

Reply:
Understood! yeah did not cross my mind we are deep in it now

Reply:
@JoeCap08055 we have to look into what is the preferred way in substrate 
here is an example
https://github.com/paritytech/substrate/blob/master/frame/membership/src/migrations/v4.rs

Reply:
There are few different kinds of migrations, like runtime etc. since we are simply translating our schema to new type it will just be a simple hook 

Example for such https://github.com/paritytech/polkadot/pull/4545